build-in-container/README.md

```diff
@@ -0,0 +1,45 @@
+# Build-in-container
+The build-in-container tool provides a developer tool to quickly build Mariner packages. It is easy-to-use, and distribution and platform agnostic. It sets up a build environment in an expedient manner using a container.
+
+Please install docker on your system before using the tool.
+
+## Usage
+The run.sh script presents these options <br />
+-t                 creates container image <br />
+-b [repo_dir]      creates container, builds Mariner and outputs to out/ <br />
```

Comment:
Should this just default to `$(pwd)`?

Reply:
pwd might not be a valid mariner repo. Default is the repo containing the tool

Reply:
I would prefer to handle that via checking that you have a `./SPECS` dir or similar, maybe print an error if its missing but allow an override flag? I feel like the assumption should be the tool runs in your current directory unless you ask it otherwise. Having it pull up the hello world specs and place those files back in that directory is very confusing if you aren't already there.